apps/emqx_durable_storage/test/emqx_ds_storage_bitfield_lts_SUITE.erl

```diff
@@ -23,6 +23,8 @@
 -include_lib("snabbkaffe/include/snabbkaffe.hrl").
 -include_lib("stdlib/include/assert.hrl").
 
+-import(emqx_utils_conv, [bin/1]).
```

Comment:
Better to avoid imports.

Reply:
Dropped in 10e78ea194e4b8f5592b198c9fff79723905985e.